stable/aws-node-termination-handler/values.yaml

```diff
@@ -81,3 +81,5 @@ serviceAccount:
 rbac:
   # rbac.pspEnabled: `true` if PodSecurityPolicy resources should be created
   pspEnabled: true
+
+dnsPolicy: "ClusterFirst"
```

Comment:
Actually, I think it's best to make the default `ClusterFirstWithHostNet`. Can you update this and the readme default in the table

Reply:
change `dnsPolicy ` default value to `ClusterFirstWithHostNet`